.github/workflows/release.yml

```diff
@@ -570,7 +572,6 @@ jobs:
             --name test-node \
             ubuntu:24.04 \
               /bin/sh -c "chmod +x /app/${{env.TEST_BIN_FILENAME}} && /app/${{env.TEST_BIN_FILENAME}} \
-              --chain=dev \
```

Comment:
Assuming this node we're launching doesn't need the funded dev accounts? (same below as well?)

Reply:
Correct. It can just start with basic genesis